### ISSUE TITLE
Bug: Side Nav mobile accordion treatment does not meet specs

### ISSUE THREAD

**Issue by erynsobing** [department-of-veterans-affairs/va.gov-team#130919]
### Description

The Side Nav mobile accordion visual treatment doesn't match the design specs from Midpoint Review. The current implementation uses a chevron icon instead of a plus/minus sign, and the blue outline treatment is missing. 

### How severe is the problem?

- [ ] Critical - A Veteran is blocked or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] Moderate - A Veteran is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [x] Minor - Cosmetic bugs and minor language inconsistencies. Defer to sprint planning.

### Desired behavior

See [Figma](https://www.figma.com/design/pvrA4dBGYhgiRyI0cxN8vE/526---Side-Navigation?node-id=4101-107368&t=PRVFJ5mbI4ADPAG6-4) for the expected design.

[See the latest mobile designs](https://www.figma.com/design/pvrA4dBGYhgiRyI0cxN8vE/526---Side-Navigation?node-id=4935-17997&t=jgaOdzVpzmsjeYNN-1) (Figma)

### Tasks

- Collaborate as needed with Platform team to meet the design specs for the Side nav component
- Update chevron
- Update blue outline treament
- Test with a Designer

### Acceptance Criteria

- [ ] Implementation matches design specs
- [ ] Reviewed/tested by a Designer
- [ ] Design team has been notified to submit the ADE team request once PR is merged

### Definition of Done

- [ ] Meets acceptance criteria
- [ ] Reviewed and approved by product and/or design

### Engineering

- [ ] All tests pass
- [ ] New functionality is covered by unit tests
- [ ] Logging and monitoring are implemented (if applicable)

### Code Review & Pull Requests

- [ ] PR includes Local testing steps
- [ ] PR includes Flipper/testing state details (if applicable)
- [ ] PR includes Author's local proof of submission screenshot
- [ ] Copilot review completed and feedback addressed
- [ ] Internal reviewer approved
- [ ] Internal reviewer added local proof of submission screenshot
- [ ] Code functionality verified on Staging after merge

### Refinement Checklist!

- [ ] Added description, tasks, and acceptance criteria
- [ ] Added estimate
- [ ] Labeled with Practice Area (engineer, design, product, data science)
- [ ] Labeled with issue type and characteristics of the ticket (bug, accessibility, request, discovery, documentation, research, content, ux testing, front-end, back-end, datadog, etc.)
- [ ] Added any other relevant project fields (team, OCTO priority...)
- [ ] Added an Epic or Super Epic

**Comment by fiorella-io** [comment]
@Dr-Pongo, I have a few edits

1. Can the "+" and "-" be updated to use a chevron?
2. Update border thickness from .2 rem to .15rem   
3. For MPV, the accordion should be placed below the progress bar. (checking with the team)
<img width="40%" src="https://github.com/user-attachments/assets/de108e8b-6cc4-46ca-a319-f83f0df7dece" /> 
<img width="40%" src="https://github.com/user-attachments/assets/ac275764-063b-4ad7-9520-7baeb1cadad8" />

## It should look like this

<img width="40%" src="https://github.com/user-attachments/assets/42e7b72b-d6d2-4ce7-a5f9-cbf5359713dc" />
